qiskit/circuit/controlledgate.py

```diff
@@ -174,10 +174,10 @@ def num_ctrl_qubits(self, num_ctrl_qubits):
         Raises:
             CircuitError: num_ctrl_qubits is not an integer in [1, num_qubits - 1].
         """
-        if num_ctrl_qubits == int(num_ctrl_qubits) and 1 <= num_ctrl_qubits < self.num_qubits:
+        if num_ctrl_qubits == int(num_ctrl_qubits) and 1 <= num_ctrl_qubits <= self.num_qubits:
```

Comment:
We should also update the docs to mention the control qubits must be in `[1, num_qubits]` 😄

Reply:
Ah, thanks - I have my editor set to keep docstrings folded by default and missed it!  Fixed in 23ea8950e.